src/XLParser/ExcelFormulaGrammar.cs

```diff
@@ -207,13 +215,13 @@ public ExcelFormulaGrammar() : base(false)
             #region Functions
 
             FunctionCall.Rule =
-                  Function + Arguments + CloseParen
+                  FunctionName + Arguments + CloseParen
                 | PrefixOp + Formula
                 | Formula + PostfixOp
                 | Formula + InfixOp + Formula
                 ;
-
-            Function.Rule = ExcelFunction | UDFToken;
+                
+            FunctionName.Rule = ExcelFunction;
```

Comment:
Considered that myself too. I've left it like this because now all function calls (normal, reference, UDF) have `NonTerminal-Name NonTerminal-Arguments \`)\`` as children, which simplifies processing ([example](https://github.com/PerfectXL/XLParser/blob/grammar/udf-as-reference/src/XLParser/ExcelFormulaParser.cs#L204)).